pkgs/applications/misc/michabo/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, writeText
+, fetchFromGitLab
+, qmake
+# qt
+, qtbase
+, qtwebsockets
+}:
+
+let
+  desktopFile = writeText "michabo.desktop" ''
+    [Desktop Entry]
+    Type=Application
+    Name=Michabo
+    Exec=Michabo
+  '';
+in stdenv.mkDerivation rec {
+  pname = "michabo";
+  version = "0.1";
+
+  src = fetchFromGitLab {
+    domain = "git.pleroma.social";
+    owner = "kaniini";
+    repo = "michabo";
+    rev = "v${version}";
+    sha256 = "0pl4ymdb36r0kwlclfjjp6b1qml3fm9ql7ag5inprny5y8vcjpzn";
+  };
+
+  nativeBuildInputs = [
+    qmake
+  ];
+  buildInputs = [
+    qtbase
+    qtwebsockets
+  ];
+
+  configurePhase = "qmake michabo.pro DESTDIR=$out/bin";
```

Comment:
What about using

```nix
  qmakeFlags = [ "michabo.pro" "DESTDIR=${placeholder "out"}/bin" ];
```

Though it looks like the shipped `.pro` file is incorrect, as it ~~ignores the `PREFIX` set in the [setup hook](https://github.com/NixOS/nixpkgs/blob/a4860c9cb2407db392615399fda54eaca7a8c71c/pkgs/development/libraries/qt-5/hooks/qmake-hook.sh#L4).~~ installs the program during build?

Reply:
Nice! didn't know about qmakeFlags and placeholder, thanks for the suggestion. I updated the commit.

I'll report the .pro problem upstream.